src/shared/utils.ts

```diff
@@ -33,3 +33,7 @@ export function isTypedArray(val: unknown): val is Uint8Array {
 
 // eslint-disable-next-line @typescript-eslint/no-empty-function
 export function noop() {}
+
+export function arrayIfNotArray<T>(value: T | T[]): T[] {
```

Comment:
wdyt?
<img width="673" alt="Screenshot 2023-05-29 at 12 33 03" src="https://github.com/hirosystems/wallet/assets/46521087/453630f1-f70c-41af-879a-a84a247b7925">

Reply:
Yes, I like this name better 👍🏼

Reply:
What was the prompt?

Reply:
`how to better name this function export function arrayIfNotArray<T>(value: T | T[]): T[] {
  return Array.isArray(value) ? value : [value];
}`